en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md

```diff
@@ -36,25 +42,39 @@ A **data service** provides a web service interface to access data that is stor
 
 Follow the steps given below to create a new data service.
 
-#### Creating a data service project
+#### Creating a Maven Multi Module Project
```

Comment:
Shouldn't we create an integration project here instead?

Reply:
MMM is created because we don't need Config modules for the data service.